examples/golang/envoy.yaml

```
@@ -0,0 +1,59 @@
+# envoy demo with golang extension enabled
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 8080
```

```suggestion
        port_value: 10000
```

as the Envoy container already exposes port 10000 its better to use this